release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -372,6 +372,15 @@ Pods created using the macvlan CNI plugin, where the IP address management CNI p
 [id="ocp-4-15-registry"]
 === Registry
 
+[id="ocp-4-15-private-storage-endpoint-azure"]
+==== Support for private storage endpoint on {azure-short}
+
+With this release, the Image Registry Operator can be leveraged to use private storage endpoints on {azure-short}. This feature enables seamless configuration of private endpoints for storage accounts when {product-title} is deployed on private {azure-short} clusters, which allows users to deploy the image registry without exposing public-facing storage endpoints.
```

Comment:
```suggestion
With this release, the Image Registry Operator can be leveraged to use private storage endpoints on {azure-short}. You can use this feature to seamlessly configure private endpoints for storage accounts when {product-title} is deployed on private {azure-short} clusters, so that users can deploy the image registry without exposing public-facing storage endpoints.
```

Per IBMSG, avoid using "allow"
![image](https://github.com/openshift/openshift-docs/assets/40172997/cb5a68a6-ade1-4626-8b91-a91c404d9bf8)

Same thing for "enable"
![image](https://github.com/openshift/openshift-docs/assets/40172997/c1026456-940d-437f-856e-d61941a82d08)

Added some suggested wording to get around those.